covid_graphs/covid_graphs/scripts/country_dashboard.py

```diff
@@ -185,6 +216,26 @@ def update_graph(prediction_event_name, graph_type_str, country_short_name):
             figure = graphs[0].create_country_figure(graph_type)
             return figure
 
+    def _create_single_country_all_predictions_callbacks(self):
+        # TODO: Predictions will be generated statically in the future.
```

Comment:
It's fine doing it at the restart (it might become a problem with 100 countries though).